mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraMailboxMapper.java

```diff
@@ -218,12 +222,17 @@ public boolean hasChildren(Mailbox mailbox, char delimiter) {
     public void updateACL(Mailbox mailbox, MailboxACL.ACLCommand mailboxACLCommand) throws MailboxException {
         CassandraId cassandraId = (CassandraId) mailbox.getMailboxId();
         cassandraACLMapper.updateACL(cassandraId, mailboxACLCommand);
+        if (mailboxACLCommand.getEntryKey().isUser()) {
```

Comment:
In my opinion, we lack an ACLChangedEvent that is triggered on ACL changes. With that event, we could handle these ACL updates (and projections building) based on events

Reply:
IMO we don't need it now. Cassandra DAO consistency is very well handled without.

Thus I would advise to do it later to avoid unecessary big refactoring